AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -142,6 +142,7 @@ - (void)updateButtonContent
   [self updateBackgroundImage];
   [self updateImage];
   [self updateTitle];
+  [self setNeedsLayout];
```

Comment:
Yep they call the update method internally, don't really know why I added it in the first place :/ but it's removed now